specs/0001-allow-disable-date-input/product-spec.md

```diff
@@ -0,0 +1,102 @@
+---
+author: @jrieke
```

Comment:
nit: I think you need to put this into quotes since the @ is causing issues when rendered on Github:

```suggestion
author: "@jrieke"
```

<img width="842" height="334" alt="Image" src="https://github.com/user-attachments/assets/cbeacdc7-be48-4587-82d1-c33112eab239" />